go/cmd/dolt/commands/fetch.go

```diff
@@ -161,7 +168,7 @@ func mapRefspecsToRemotes(refSpecs []ref.RemoteRefSpec, dEnv *env.DoltEnv) (map[
 	return rsToRem, nil
 }
 
-func fetchRefSpecs(ctx context.Context, dEnv *env.DoltEnv, rem env.Remote, refSpecs []ref.RemoteRefSpec) errhand.VerboseError {
+func fetchRefSpecs(ctx context.Context, forceFetch bool, dEnv *env.DoltEnv, rem env.Remote, refSpecs []ref.RemoteRefSpec) errhand.VerboseError {
```

Comment:
unideal naked boolean. I could leave it, pass the argParser, or reorganize everything to create a separate forceFetch path. 
Thoughts?

Reply:
Can also technically do something like:

```go
type RefUpdateMode struct {
        bool
}
var ForceUpdate = RefUpdateMode{true}
var FastForwardOnly = RefUpdateMode{false}

func main() {
        a(ForceUpdate)
        a(FastForwardOnly)
}

func a(mode RefUpdateMode) {
        fmt.Println(mode.bool)
}
```

Reply:
I think naked boolean is fine here for now. This method isn't public, for one...